src/core/bpf-lsm.c

```diff
@@ -0,0 +1,335 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
```

Comment:
please sort the includes - system headers in the first block, internal headers in the second block. Both blocks sorted alphabetically.